src/main/java/net/snowflake/client/core/SFTrustManager.java

```diff
@@ -1163,7 +1161,7 @@ private OCSPResp fetchOcspResponse(
           new DecorrelatedJitterBackoff(sleepTime, MAX_SLEEPING_TIME_IN_MILLISECONDS);
       boolean success = false;
 
-      final int maxRetryCounter = isOCSPFailOpen() ? 1 : 3;
+      final int maxRetryCounter = isOCSPFailOpen() ? 1 : 2;
```

Comment:
@sfc-gh-pfus should we change retry counter?

Reply:
Yes, it's fine.